projects/mysadna/src/app/services/data.service.ts

```diff
@@ -3,45 +3,13 @@ import { Observable, of } from 'rxjs';
 import { HttpClient, HttpHeaders } from '@angular/common/http';
 import { catchError, map, tap } from 'rxjs/operators';
 
-import { User } from '../models/user';
-import { Project } from '../models/project';
+import { User } from '@/models/user';
+import { Project } from '@/models/project';
+import { USERS } from '@/mock-data/mock-users';
+import { PROJECTS } from '@/mock-data/mock-projects';
 
-import {USERS} from '../mock-data/mock-users';
-import {PROJECTS} from '../mock-data/mock-projects';
-
-const httpOptions = {
-  headers: new HttpHeaders({ 'Content-Type': 'application.json' })
-};
-
-@Injectable({
-  providedIn: 'root'
-})
+@Injectable()
 export class DataService {
-  // private usersUrl = 'api/users'; //For web api
-  // private projectsUrl = 'api/projects';
```

Comment:
garbage code

Reply:
I guess I should also get rid of the two unused imports, then?
`import { HttpClient, HttpHeaders } from '@angular/common/http';
import { catchError, map, tap } from 'rxjs/operators';`